### ISSUE TITLE
Unable to add Microsoft.EntityFrameworkCore.InMemory v1.0.0 to UWP project

### ISSUE THREAD

**Issue by hameedfs** [dotnet/efcore#5932]
When I add Microsoft.EntityFrameworkCore.InMemory v1.0.0 NuGet package to UWP app project I get the below error. I am using VS2015 update 3. The error occurs even when adding the NuGet package to a new blank app.

**System.Reflection.Emit.ILGeneration 4.0.1 provides a compile-time reference assembly for System.Reflection.Emit.ILGeneration on UAP,Version=v10.0, but there is no run-time assembly compatible with win10-x86-aot.**

The full output of package manager is attached.

[PackageManagerOutput.txt](https://github.com/aspnet/EntityFramework/files/343431/PackageManagerOutput.txt)

**Comment by natemcmaster** [comment]
Your log output shows downgrade warnings, which are the tip of the iceberg. This is caused by using an old version of Microsoft.NETCore.UniversalWindowsPlatform. Upgrade this to 5.2.0. (Be warned: the .NET team decided to unlist it so this version won't appear in NuGet UI #5902).
```json
{
  "dependencies": {
    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.0",
    "Microsoft.EntityFrameworkCore.InMemory": "1.0.0"
  },
  "frameworks": {
    "uap10.0": {}
  },
  "runtimes": {
    "win10-arm": {},
    "win10-arm-aot": {},
    "win10-x86": {},
    "win10-x86-aot": {},
    "win10-x64": {},
    "win10-x64-aot": {}
  }
}
```

**Comment by ericstj** [comment]
Just to update this thread, the new published version is 5.2.2.  This should be used in UWP whenever working with packages that use netstandard.
